functional/util/config.go

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
Please switch to line comments, we just did this elsewhere (block comments interfere with build tags :/)